### DISCUSSION TITLE
这个启动好后，nginx是需要单独安装吗？没太看懂这个，还是安装出错，启动也始终失败

### DISCUSSION THREAD

**Discussion by StoneNBS** [0xJacky/nginx-ui#188] [Q&A]
![image](https://github.com/0xJacky/nginx-ui/assets/14786494/8c8ff2ef-80ff-4c4c-98c6-9b7f20089ad1)
![image](https://github.com/0xJacky/nginx-ui/assets/14786494/a9b40b36-44c1-486e-acee-579faec9a11d)

**Comment by 0xJacky** [comment]
是的 nginx 要单独安装

**Comment by 0xJacky** [comment]
docker 版本是集成了 nginx 的

**Comment by StoneNBS** [comment]
![image](https://github.com/0xJacky/nginx-ui/assets/14786494/d63dffa6-3891-49cd-8451-3b9a33ba1717)

**Comment by StoneNBS** [comment]
启动好后，好像还连接github，我们国内连接会很慢

**Comment by 0xJacky** [comment]
> 启动好后，好像还连接github，我们国内连接会很慢

启动后不会连接 Github，除了下载更新，这个错误不是因为网络引起的.....

参考：
https://nginxui.com/zh_CN/guide/config-nginx-log.html

**Comment by 0xJacky** [comment]
注意，Nginx UI 不包括 Nginx，请阅读 https://nginxui.com/zh_CN/guide/getting-started.html

**Comment by StoneNBS** [comment]
感觉好像搞不起来，有没有能远程看下

**Comment by Hintay** [comment]
抱歉，我们无法提供任何远程操作。此外，此类非 BUG 相关的问题请于 Discussions 中讨论。
